### DISCUSSION TITLE
Webservice DNS hijacked?

### DISCUSSION THREAD

**Discussion by omerta324** [spatie/ssl-certificate-chain-resolver#47] [Q&A]
Hi all - it seems that the webservice (https://certificatechain.io/) for this tool has been hijacked? It directs me to a wedding party.

**Comment by freekmurze** [comment]
We don't operate that site anymore, I've removed it from the readme now.

**Reply by nunofranciscomoreira** [reply]
Just a FYI, I believe you forgot one reference as it still shows up here. Check picture attached.
![IMG_20221022_163151](https://user-images.githubusercontent.com/24529260/197347796-70a1f791-8dc7-414a-b763-813068fc4e88.jpg)
